docs/userguide/products/fault-association.html

```diff
@@ -0,0 +1,242 @@
+
+                <dt>version</dt>
+                <dd>event version given by the network operator</dd>
+
+                </dd>
+        </dl>
+
+<h3>Fault Feature</h3>
+<dl>    
+                <dt>id</dt>
+                <dd>SCEC CFM fault id</dd>
+  
+</dl>
+<h4>Fault Feature properties</h4>
+<dl>
+		<dt>distance</dt>
+		<dd>distance from epicenter in kilometers</dd>
+
+		<dt>probability</dt>
+		<dd>The probability in percent the earthquake is associated with this fault</dd>
+
+		<dt>name</dt>
+		<dd>SCEC CFM 5.0 Fault name and closest segment if available.</dd>
+
+		<dt>qfaultids</dt>
+		<dd>an array of corresponding faults ids in the QFault database.</dd>
+
+		</dd>
+	</dl>
+
+<h3>Example</h3>
+<div class="code"><pre> {
+  "metadata": {
+    "process_time": "2021-04-13T21:40:03Z",
+    "software_version": "NA",
+    "cfm_version": "5"
+    },
+  "result": [
+    {
+      "probability": 37,
+      "title": "Not in CFM"
+    },
+    {
+      "probability": 35,
+      "title": "Berdoo Canyon fault"
+    },
+    {
+      "probability": 28,
+      "title": "Other CFM faults"
+    }
+  ],
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "properties": {
+        "class": "event",
+        "mag": 3.4,
+        "utctime": "2021-04-13T21:35:03",
+        "title": "M 3.5 - 15km NE of Indio, CA",
+        "version": "5"
+      },
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          -116.09,
+          33.807,
+          2.7
+        ]
+      },
+     "id": "ci39849144"
+    },
+    {
+      "type": "Feature",
+      "properties": {
+        "class": "fault",
+        "name": "Berdoo Canyon fault",
+        "qfaultids": [
+          "12"
+        ],
+        "distance": 7.12,
+        "probability": 35
+      },
+      "id": "213",
+      "geometry": null
+    },
+    {
+      "type": "Feature",
+      "properties": {
+        "class": "fault",
+        "name": "Mecca Hills; Hidden Springs fault segment",
+        "qfaultids": [
+          "97",
+          "265",
+          "313",
+          "316",
+          "314",
+          "297",
+          "265",
+          "313",
+          "316",
+          "314"
+        ],
+        "distance": 7.82,
+        "probability": 10
+      },
+      "id": "218",
+      "geometry": null
+    },
+    {
+      "type": "Feature",
+      "properties": {
+        "class": "fault",
+        "name": "Indio Hills fault",
+        "qfaultids": [
+          "297"
+        ],
+        "distance": 8.97,
+        "probability": 7
+      },
+      "id": "214",
+      "geometry": null
+    },
+    {
+      "type": "Feature",
+      "properties": {
+        "class": "fault",
+        "name": "Blue Cut fault ",
+        "qfaultids": [
+          "30"
+        ],
+        "distance": 10.65,
+        "probability": 6
+      },
+      "id": "30",
+      "geometry": null
+    }
+  ]
+}</pre></div>
+
+--------------------
+<h2>Contents</h2>
```

Comment:
There should be one contents list, briefly describing all contents, and then you can add example sections at the end of the page.